solutions/swb-app/src/datasetRoutes.ts

```diff
@@ -10,7 +10,7 @@ import {
   DataSetService,
   DataSetsStoragePlugin
 } from '@aws/workbench-core-datasets';
-import Boom from '@hapi/boom';
+import * as Boom from '@hapi/boom';
```

Comment:
why did eslint want this fix?

Reply:
Using exported name 'Boom' as identifier for default export. eslint [import/no-named-as-default](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/no-named-as-default.md)

Reply:
That feels a little weird but 🤷